### PR TITLE
Improvements to README

### PR DESCRIPTION
**Pull request by aggarw13**
* Separate sections of building library and building unit tests. 
* Clearly mention about use of the `mqttFilePaths.cmake` file for building library﻿

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,21 +17,31 @@ and build the library with default configuration values, provide `MQTT_DO_NOT_US
 **OR**
 * Defining the `MQTT_DO_NOT_USE_CUSTOM_CONFIG` preprocessor macro for the library build.
 
+## Building Library
+
+The [mqttFilePaths.cmake](https://github.com/FreeRTOS/coreMQTT/blob/master/mqttFilePaths.cmake) file contains the information of all source files and the header include path required to build the NQTT library.
```
Comment (sarenameas) [inline README.md:22]: MQTT

File: README.md [hunk 2]
```diff
@@ -19,21 +17,31 @@ and build the library with default configuration values, provide `MQTT_DO_NOT_US
 **OR**
 * Defining the `MQTT_DO_NOT_USE_CUSTOM_CONFIG` preprocessor macro for the library build.
 
+## Building Library
```
Comment (sarenameas) [inline README.md:20]: Building the Library

Comment (aggarw13) [reply README.md]: Updated

**Comment by muneebahmed10** [comment]
Can you also update the link to the hub repo, since it shouldn't point to the development branch any more? https://github.com/FreeRTOS/coreMQTT/blame/master/README.md#L42

**Review by sarenameas — APPROVED** [review]

**Review by n9wxu — APPROVED** [review]

**Pull request merged**